native/effektio/src/platform/mod.rs

```diff
@@ -0,0 +1,17 @@
+#[cfg(target_os = "android")]
+mod android;
+
+#[cfg(target_os = "android")]
+pub use android::*;
+
+#[cfg(target_os = "ios")]
+mod ios;
+
+#[cfg(target_os = "ios")]
+pub use ios::*;
+
+#[cfg(not(any(target_os = "android", target_os = "ios")))]
+mod test_platform;
+
+#[cfg(not(any(target_os = "android", target_os = "ios")))]
+pub use test_platform::*;
```

Comment:
could you elaborate the use of test_platform? why not just go for `native` as a real target here?

Reply:
I'll rename it

Reply:
done